src/ripple/app/ledger/tests/Ledger_test.cpp

```diff
@@ -44,14 +44,13 @@ class Ledger_test : public beast::unit_test::suite
         auto mark = createAccount ("mark", keyType);
 
         // Fund gw1, gw2, gw3, alice, mark from master
-        makeAndApplyPayment(master, gw1, 5000 * xrp, ledger, sign);
-        makeAndApplyPayment(master, gw2, 4000 * xrp, ledger, sign);
-        makeAndApplyPayment(master, gw3, 3000 * xrp, ledger, sign);
-        makeAndApplyPayment(master, alice, 2000 * xrp, ledger, sign);
-        makeAndApplyPayment(master, mark, 1000 * xrp, ledger, sign);
+        makePayment(master, gw1, 5000 * xrp, ledger, sign);
```

Comment:
Yeah, that works. I'll also rename 'makeTrustSet` to `trust`, since that's the only reason I used the "make" naming pattern at all.